### ISSUE TITLE
Path to openmsx command seems Mac-specific

### ISSUE THREAD

**Issue by 0cjs** [farique1/MSX-Basic-Tokenizer#1]
Running `openMSXbatoken.py` on Linux, I get the following error (I've elided some of the path with `…`):

    FileNotFoundError: [Errno 2] No such file or directory: '…/MSX-Basic-Tokenizer/openMSX.app/contents/macos/openmsx'

Looking at the code, I see that your command is hardcoded:

    cmd = [openmsx_filepath + '/contents/macos/openmsx', '-control', 'stdio']

Doing a quick hack to just run `openmsx` and let it be found from the path:

    cmd = ['openmsx', '-control', 'stdio']

I find that it now successfully runs `openmsx`. (On my system, where the Debian `openmsx` package is installed, this is in `/usr/bin/openmsx`, but if I wanted to run a more recent version of openMSX I'd probably build it myself and put the binary or a symlink in `~/.local/bin/openmsx`, or perhaps `~/.local/bin/openmsx-0.14.0` or something like that if I were using several versions.)

I suggest that the code be changed so that the user specifies in the configuration a full path to the `openmsx` command (including the filename itself, not just a directory) and, if that's not given, just `openmsx` be used. This should make it work by default on many platforms, and remove the need for users to create what look like Mac-specific paths on non-Mac machines that then symlink (or do who-knows-what on Windows) to their actual openMSX binary.

**Comment by farique1** [comment]
You are right and this is on the TODO list for all the programs related to the MSX Basic Dignified suit.
Giving control of the file name and using the path.join() method is the way to go for a more complete portability.
I don't have a Windows or Linux machine to test but the implementation is straightforward enough.
Now I just have to find some time.
Thanks for the comment.
